lib/ansible/modules/windows/win_power_plan.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/python
+# Copyright (c) 2017 Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.0',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+---
+module: win_power_plan
+short_description: Changes the power plan of a Windows system
+description:
+  - win_power_plan will change the power plan of a Windows system to the defined string.
+  - Windows defaults to "balanced" which will cause CPU throttling. In some cases it can be preferable
```

Comment:
You may want to use C(balanced) and C(high performance) to make it stand out more in the documentation.